r2dbc-mysql/src/test/java/io/asyncer/r2dbc/mysql/MySqlTypeMetadataTest.java

```diff
@@ -50,11 +52,24 @@ void noSet() {
 
     @Test
     void isBinaryUsesCollationId() {
-        MySqlTypeMetadata metadata = new MySqlTypeMetadata(0, -1, CharCollation.BINARY_ID);
+        MySqlTypeMetadata metadata = new MySqlTypeMetadata(0, -1, CharCollation.BINARY_ID, null);
 
         assertThat(metadata.isBinary()).isTrue();
 
-        metadata = new MySqlTypeMetadata(0, -1, 33);
+        metadata = new MySqlTypeMetadata(0, -1, 33, null);
         assertThat(metadata.isBinary()).isFalse();
     }
+    
+    @Test
```

Comment:
Could you add an integration test as well? to make it sure?

Reply:
Hi @jchrys ,
I've added an integration test in MariaDbIntegrationTestSupport but I'm unsure of how to set/enable the extended type info capability for the test server. Do you know how to set this so that the test won't fail?

Reply:
Hello, @svats0001.
Extended type info was introduced in mariadb 10.5. I believe adding `@EnabledIf("envIsMariaDb10_5_1")` would accommodate this.

Reply:
Hmm, seems like the database isn't returning the extended type information @jchrys . Is there some flag that we need to set for the database to return this info or should it automatically return this after version 10.5 (this is what I assume to be true)?